src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -22,6 +22,10 @@
      */
     String getDescription();
 
+    default String getCSSName(){
```

Comment:
Operation should not have anything to do with CSS, since it's not in the gui package.